scripts/world/Player.gd

```diff
@@ -0,0 +1,28 @@
+extends KinematicBody2D
+
+const SPEED = 200
+
+var motion = Vector2()
+
+func _physics_process(delta):
+    var go_left = Input.is_action_pressed("ui_left")
+    var go_right = Input.is_action_pressed("ui_right")
+    var go_up = Input.is_action_pressed("ui_up")
+    var go_down = Input.is_action_pressed("ui_down")
+
+    if go_left:
+        motion.x = -SPEED
+
+    if go_right:
+        motion.x = SPEED
+
+    if go_up:
+        motion.y = -SPEED
+
+    if go_down:
+        motion.y = SPEED
+
+    motion.x = lerp(motion.x, 0, 0.2)
```

Comment:
I just realized that delta should be present in these `lerp`s to make them framerate independent. I will create an issue for this to fix in both player scripts.

Reply:
Honestly, I still don't understand what this does and I've just pasted it from the other Player code. I won't touch it for now and we can fix it in the other issue.